includes/Modules/Analytics.php

```diff
@@ -481,6 +488,7 @@ protected function get_datapoint_services() {
 			'report'                       => 'analyticsreporting',
 			// POST.
 			'settings'                     => '',
+			'create-account-ticket'        => 'analytics',
```

Comment:
This needs to use the custom `analyticsprovisioning` instead.

Reply:
fixed.